src/quo2/components/loaders/community_card_skeleton/style.cljs

```diff
@@ -0,0 +1,95 @@
+(ns quo2.components.loaders.community-card-skeleton.style
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.foundations.shadows :as shadows]))
+
+(defn card
+  [width theme]
+  (merge
+   {:width            width
+    :height           230
+    :border-radius    16
+    :background-color (colors/theme-colors colors/neutral-10 colors/neutral-80 theme)}
+   (shadows/get 2)))
```

Comment:
@vkjr, similar to `colors/theme-colors`, `shadows/get` accepts a theme argument. This is necessary the majority of the time and it's necessary here.

Recently, when I rewrote the `shadows` namespace I even thought of not having a fallback to the global ratom `quo2.theme/theme-state` because it's more "bug-driven" because devs need to constantly remember to pass the theme argument to make the components truly themeable.

Reply:
@ilmotta, thanks for noticing!